examples/with-cloud/app/api/assistant-ui-token/route.ts

```
@@ -0,0 +1,55 @@
+import { customAlphabet } from "nanoid";
+import { cookies } from "next/headers";
+import jwt, { JwtPayload } from "jsonwebtoken";
+import { AssistantCloud } from "@assistant-ui/react";
+
+const generateId = customAlphabet(
+  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz",
+  32,
+);
+const randomUserId = () => {
+  const userId = "usr_anon_" + generateId();
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)